packages/sdk/src/swap/types.ts

```diff
@@ -34,19 +34,19 @@ interface ChainsAndAssets {
 
 export interface QuoteRequest extends ChainsAndAssets {
   amount: string;
-  boostFeeBps?: number;
   brokerCommissionBps?: number;
   affiliateBrokers?: AffiliateBroker[];
 }
 
 export interface QuoteResponse
-  extends Omit<QuoteRequest, 'boostFeeBps' | 'brokerCommissionBps' | 'affiliateBrokers'> {
+  extends Omit<QuoteRequest, 'brokerCommissionBps' | 'affiliateBrokers'> {
   quote: QuoteQueryResponse;
 }
 
 export interface DepositAddressRequest extends QuoteRequest {
   destAddress: string;
   ccmMetadata?: CcmMetadata;
+  boostFeeBps?: number;
```

Comment:
shouldn't `boostFeeBps` be part of the DepositAddressResponse?

Reply:
It is under `DepositAddressRequest` because it is also used when opening a deposit channel. 
The `DepositAddressRequest` extends from `QuoteRequest` and the difference here (compared to the broker comission) is that we decided to not support passing a `boostFeeBps` as an argument when fetching a quote but instead calculate the effective boost fee automatically.